src/lib/identinet/mod.js

```diff
@@ -33,6 +33,7 @@ export function url2DID(url) {
 }
 
 export async function getCurrentTab() {
+  if (!api) return [];
```

Comment:
changes in this file are necessary to get everything running in browser (where chrome and browser vars are undefined). @jceb could you check if this breaks the default build process / behavior?